server.js

```diff
@@ -2,153 +2,22 @@
 
 require('dotenv').config();
 
-const express = require('express');
-const bodyParser = require('body-parser');
-const session = require('express-session');
-const passport = require('passport');
-const saml = require('passport-saml');
-const { ApolloServer } = require('apollo-server-express');
-const { Op } = require('sequelize');
-const db = require('./models');
-const { schema } = require('./graphql');
+const App = require('./app');
 
 const PORT = 4000;
 const HOST = `http://localhost:${PORT}`;
```

Comment:
should this HOST be specified here? would it make more sense to specify it in `app.js`, since I see that the only place we're using this is to console log the `App ready on ${HOST}`

Reply:
we don't use `HOST` in app either so it's not necessary - to be honest it's not really doing anything and only handy for development to ensure that your server spun up; overall let's just leave it in